Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -251,11 +251,10 @@ final class StatusTableViewController: LoopChartsTableViewController {
     }
 
     private func registerPumpManager() {
-        if let pumpManager = deviceManager.pumpManager {
-            self.basalDeliveryState = pumpManager.status.basalDeliveryState
-            pumpManager.removeStatusObserver(self)
-            pumpManager.addStatusObserver(self, queue: .main)
-        }
+        self.basalDeliveryState = deviceManager.pumpManager?.status.basalDeliveryState
+        self.bolusState = deviceManager.pumpManager?.status.bolusState ?? .none
```

Comment:
Shouldn't the `bolusState` also be an optional set to nil when there is no pod? It is a different state than `.none` (which means there is a pod, but no current ongoing bolus happening).